adagrams/game.py

```diff
@@ -1,11 +1,141 @@
+import random
+from random import choices
+import copy
+
+LETTER_DICT = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+}
+
 def draw_letters():
-    pass
+
+    letter_bank = []
+    letter_copy = copy.deepcopy(LETTER_DICT)
+
+    while len(letter_bank) < 10:
+        key_list = list(letter_copy.keys())
+        random_letter = random.choice(key_list)
+        if letter_copy[random_letter] > 0:
+            letter_bank.append(random_letter)
+            letter_copy[random_letter] -= 1
```

Comment:
Nice job using the helper dictionary to decrement the value in the letter pool to keep track of how many letters have been played!